onnxruntime/test/testdata/onnx_backend_test_series_filters.jsonc

```diff
@@ -61,7 +61,15 @@
         "^test_mul_uint8_cpu",
         "^test_add_uint8_cpu",
         "^test_div_uint8_cpu",
-        "^test_batchnorm_*"
+        "^test_batchnorm_*",
```

Comment:
can you change "^test_batchnorm_*" to : 
"^test_batchnorm_epsilon_training_mode_cuda",
"^test_batchnorm_example_training_mode_cuda",